pkg/api/cluster_types.go

```diff
@@ -193,6 +197,12 @@ type StrimziVersion struct {
 	KafkaIBPVersions []KafkaIBPVersion `json:"kafkaIBPVersions"`
 }
 
+type DynamicCapacityInfo struct {
+	MaxNodes       int32 `json:"max_nodes"`
+	MaxUnits       int32 `json:"max_units"`
```

Comment:
Can we be more specific here too? I understand we are talking about `Streaming Units`

Reply:
Correct. This is referring to streaming units. Do you think adding comments to these fields might be enough?

Reply:
If possible I would like to be more specific for all of them. Specially for the nodes one

Reply:
I think it is possible. I'll add the comments. 
The reason why I do not want to change the Field name is that the values are already being used in the Fleetshard. 
we could change them for our internal stuffs and not touch the exposed API but that might introduce some inconsistency. 
I'll open a follow PR to add comments to the fields.

Reply:
Thanks for raising this!

Reply:
I've opened https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1151

Reply:
:+1: